Add tool guide, getting started, python tutorial (#324)
terra-project/docs: docs/develop/tools.md

# Which tools to use <img src="/img/build_a_dApp_ver2.svg" height="40px">

Terra provides a variety of tools for builders that make up the Terra Development Suite. Use this guide to find the right tool for you. 

> If you want to know where to start, visit the [beginner's guide](get-started). 

Reviewer: ```suggestion
If you are new to Terra, start with the [beginner's guide](get-started). 
```

> ``` python

Reviewer: I like this tree alot - super helpful visual!

Terra Development Suite
│
├── Terrain: The all-in-one for smart contract development. Start here. 
│
├── LocalTerra: Local testing environment. 
│
├── Terra.js: Javascript SDK.
│
├── Terra.py: Python SDK.
│
├── Wallet Provider: Front end with templates.
│
└── Other tools
    │
    ├── terrad: Node daemon and CLI.
    │
    ├── Finder: Block explorer.
    │ 
    └── Faucet: Get testnet funds.
```

## Terrain

> Terra’s all-in-one development environment, Terrain will help you build your dApp, develop smart contracts, and interact with the Terra blockchain. Terrain provides both front and back end services for your dApp. Get started in 5 minutes with the [quickstart guide]([https://docs.terra.money/docs/develop/dapp/quick-start/README.html](https://docs.terra.money/docs/develop/dapp/quick-start/README.html))

Reviewer: ```suggestion
Terra’s all-in-one development environment, Terrain provides front-end and smart contract scaffolding for your dApp, so that you can begin developing on Terra. Get started in 5 minutes with the [quickstart guide]([https://docs.terra.money/docs/develop/dapp/quick-start/README.html](https://docs.terra.money/docs/develop/dapp/quick-start/README.html))
```

Reviewer: I think a period is also needed here

## LocalTerra

[https://github.com/terra-money/LocalTerra](https://github.com/terra-money/LocalTerra)
Set up your own blockchain testnet in less than one minute. LocalTerra is a containerized version of the Terra blockchain, complete with a functional validator set and block creation. Visit the [LocalTerra guide](./dapp/quick-start/using-terrain-localterra.md) to get started

## Terra<span/>.js

If you are familiar with Javascript, use Terra<span/>.js to create bots, power NFT mints, and for all-purpose back end services. Follow the [Terra.js tutorial](./sdks/terra-js/getting-started.md) to get started.

## Terra<span/>.py

>  If you are a Python developer, use Terra<span/>.py for on-chain analytics, Jupyter Notebooks, data processing, and other mathematical processes best suited for Python. Follow the [Terra.py tutorial](./sdks/terra-py/README.md) to get started.

Reviewer: ```suggestion
## Terra<span/>.js

Use Terra<span/>.js to create bots, power NFT mints, and for all-purpose back end services. Follow the [Terra.js tutorial](./sdks/terra-js/getting-started.md) to get started.

## Terra<span/>.py

Use Terra<span/>.py for on-chain analytics, Jupyter Notebooks, data processing, and other mathematical processes best suited for Python. Follow the [Terra.py tutorial](./sdks/terra-py/README.md) to get started.
```

Ditto my comments above
